### ISSUE TITLE
Share Access with a user with no verified e-mail link overflows past modal

### ISSUE THREAD

**Issue by simonv3** [sandstorm-io/sandstorm#2408]
[bite-size]

![screenshot 2016-08-17 16 42 39](https://cloud.githubusercontent.com/assets/485583/17758260/5223346a-649a-11e6-9be6-a8bf3f601753.png)

This can probably be fixed with some CSS

**Comment by kentonv** [comment]
FWIW this should almost never happen in the real world since all real users should have e-mail addresses.

**Comment by simonv3** [comment]
Yeah, this happened on the local.sandstorm.io:6080 instance while testing.
